rustfmt-core/rustfmt-lib/src/comment.rs

```diff
@@ -528,7 +528,6 @@ impl<'a> CommentRewrite<'a> {
             .checked_sub(closer.len() + opener.len())
             .unwrap_or(1);
         let indent_str = shape.indent.to_string_with_newline(config).to_string();
-        let fmt_indent = shape.indent + (opener.len() - line_start.len());
```

Comment:
I have this nagging suspicion that the offset calc here was used to handle some edge case, though for the life of me I can't think of what that may be nor have I been able to come up with any kind of input that breaks because of this change :man_shrugging: 

As such I think this is probably a good fix, though I wouldn't be completely shocked to see an issue reported at some point in the future

Reply:
Yes agreed that there's some risk here to extant code that lacks corresponding tests. I *think* this logic may have had to do with some confusion in the code about whether `*` or `**` was the right line continuation on doc block comments.

I looked around for C-style comments in a bunch of rust code. I found very little, but didn't see issues with the code I did find.